portal-api/src/v1/users/routes.test.js

```diff
@@ -11,6 +11,17 @@ jest.mock('./login.controller', () => ({
   login: () => Promise.resolve({ tokenObject: {}, user: {} }),
   sendSignInLinkEmail: jest.fn(),
 }));
+jest.mock('./authentication-email.controller', () => ({
+  validateSignInLinkToken: jest.fn(),
+}));
+jest.mock('./sanitizeUserData', () => ({
+  sanitizeUser: (user) => user,
+}));
+
+const { validateSignInLinkToken } = require('./authentication-email.controller');
```

Comment:
This path doesn't exist

Reply:
Good spot, this logic has moved/changed now anyway.